app/views/layouts/_logo.html.erb

```diff
@@ -1,8 +1,8 @@
-<a href="/" class="<%= "site-logo" unless FeatureFlag.enabled?(:creator_onboarding) %>" aria-label="<%= t("views.main.aria_home") %>">
-  <% if Settings::General.logo_svg.present? %>
+<a href="<%= root_path %>" class="site-logo" aria-label="<%= t("views.main.aria_home", community: community_name) %>">
+  <% if FeatureFlag.enabled?(:creator_onboarding) %>
+      <img class="site-logo__img" src="<%= Settings::General.resized_logo %>" alt="<%= community_name %>">
```

Comment:
Absolute nitpick but this line seems to be indented too much:

```suggestion
    <img class="site-logo__img" src="<%= Settings::General.resized_logo %>" alt="<%= community_name %>">
```

Reply:
whoops!